documentation/releaseNotes/releaseNotes.md

```diff
@@ -3,4 +3,8 @@ Today we are releasing the 6.2.0 build of the `dotnet monitor` tool. This releas
 - Allow for pushing a message to a queue when writing to Azure egress (#163)
 - Allow for simplified process filter configuration (#636)
 - Allow `config show` command to list configuration sources (#277)
-- Added Collection Rule Defaults (#1595)
+- Added collection rule defaults (#1595)
+- Added collection rule templates (#1921)
+- Added CPU usage, GC heap size, and threadpool queue length triggers (#1911)
+- Show warning when using collection rules in `connect` mode (#1844)
+- Prevent auth configuration warnings when using `--no-auth` switch (#1845)
```

Comment:
I'm aiming to get the MSI work here too.

Reply:
Sure, I'll add it in once you merge #1884